js/yarn.lock

```diff
@@ -3615,7 +3604,7 @@ ts-loader@^5.3.3:
   dependencies:
     chalk "^2.3.0"
     enhanced-resolve "^4.0.0"
-    loader-utils "^3.2.1"
+    loader-utils "^1.0.2"

```

Comment:
downgrade?

Reply:
yes, basically this upgrade impacted, that old version is a dependency.